src/aiperf/dataset/loader/mooncake_trace.py

```
@@ -85,200 +51,30 @@ def can_load(
+            data[session_id].append(trace)
+        return dict(data)
+
+    # ------------------------------------------------------------------
+    # Synthesis hooks
+    # ------------------------------------------------------------------
+
+    def _synthesis_exclude_fields(self) -> frozenset[str]:
+        return frozenset({"type"})
+
+    def _reconstruct_traces(
+        self, originals: list[MooncakeTrace], synth_dicts: list[dict[str, Any]]
```

Ruff (0.15.2)

[warning] Unused method argument: `originals`

(ARG002)